docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -39,13 +56,21 @@ Cause Model Diagram
 
 Model Assumptions and Limitations
 ---------------------------------
+This model is designed to be used for estimating DALYs due to LRI that are averted from a
+country-level intervention(e.g. food fortification or supplementation given to a percentage of the population)
+that can reduce IRI incidence as a downstream effect.
+
 There is substantial additional effort in GBD to divide LRI
-burden into the aetiologies of LRI, but we have not included
+burden into the aetiologies of LRI, but for the use of this model, we do not included
 aetiologies in this simple model.
```

Comment:
With the new wording, "included" should change to "include" to be grammatical. Also, it now sounds redundant because it says both "for the use of this model" and "in this simple model," so maybe re-word to only mention the model once.